src/containers/StructurePage/resourceComponents/AllResourcesGroup.tsx

```diff
@@ -9,47 +9,73 @@
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Plus } from '@ndla/icons/action';
+import Tooltip from '@ndla/tooltip';
+import compact from 'lodash/compact';
 import { ResourceWithNodeConnectionAndMeta } from './StructureResources';
 import { ResourceType } from '../../../modules/taxonomy/taxonomyApiInterfaces';
-import Accordion from '../../../components/Accordion';
 import ResourceItems from './ResourceItems';
-import AddResourceButton from './AddResourceButton';
 import AddResourceModal from './AddResourceModal';
+import { ChildNodeType } from '../../../modules/nodes/nodeApiTypes';
+import Resource from './Resource';
+import { NodeResourceMeta } from '../../../modules/nodes/nodeQueries';
+import ResourceBanner from './ResourceBanner';
+import { Dictionary } from '../../../interfaces';
+import AddResourceButton from './AddResourceButton';
+import { getIdFromUrn, groupResourcesByType } from '../../../util/taxonomyHelpers';
 
 interface Props {
-  currentNodeId: string;
   nodeResources: ResourceWithNodeConnectionAndMeta[];
   resourceTypes: ResourceType[];
+  currentNode: ChildNodeType;
+  contentMeta: Dictionary<NodeResourceMeta>;
+  grouped: boolean;
 }
 
-const AllResourcesGroup = ({ resourceTypes, nodeResources, currentNodeId }: Props) => {
+const AllResourcesGroup = ({
```

Comment:
Hvis vi skal bruke samme ressurs for grupperte og ugruperte ressurser foreslår jeg at vi finner et nytt navn

Reply:
Førstnevnte er sikkert fin, om ikke litt generisk. Regner med at det går fint :D